RELEASE.md

```diff
@@ -1,3 +1,26 @@
+# Release 1.3.0
+
+## New features
+
+- New E5 models added to model registry (https://github.com/marqo-ai/marqo/pull/568). E5 V2 and Multilingual E5 models are now available for use. The new E5 V2 models outperform their E5 counterparts in the BEIR benchmark, as seen [here](https://github.com/microsoft/unilm/tree/master/e5#english-pre-trained-models). See all available models [here](https://marqo.pages.dev/1.2.0/Models-Reference/dense_retrieval/).
+- Dockerfile optimisation (https://github.com/marqo-ai/marqo/pull/569). A pre-built Marqo base image results in reduced image layers and increased build speed, meaning faster docker pulls and an overall better development experience.
```

Comment:
not necessarily faster docker pulls, just neater pulls (less layers), unless you have observed this?